mpqp/core/instruction/gates/native_gates.py

```diff
@@ -1,19 +1,28 @@
 """Native gates is the set of all gates natively supported in OpenQASM. Since we
 rely on this standard, all of them are indeed implemented. In addition, this
 module contains a few abstract classes used to factorize the behaviors common to
-a lot of gates. Feel free to use them for your own custom gates!"""
+a lot of gates.
+
+You will find bellow the list of available native gates:
+
+.. container::
+    :name: native-gates-list
+    
+    ``to-be-generated``
```

Comment:
Should we remove this ?

Reply:
no, it is a target container for the dynamically generated gate list. the list is generated in js, so this stays like this until someone loads in in there browser. At that point, the `to-be-generated` is replaced by said list